e2e/rook-test-framework/rook-infra-manager/scripts/rook-dind-cluster-v1.6.sh

```diff
@@ -0,0 +1,1045 @@
+#!/bin/bash
+# Copyright 2017 Mirantis
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
if we've made any changes to this file then we should state them in the top licensing section